### ISSUE TITLE
make all -> No such file or directory~/edirect/pytest

### ISSUE THREAD

**Issue by turbomam** [linkml/schemasheets#11]
```shell
% make all
```

> poetry run pytest
/usr/local/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
Creating virtualenv schemasheets-FMUhH2LU-py3.9 in /Users/MAM/Library/Caches/pypoetry/virtualenvs

>  FileNotFoundError

>  [Errno 2] No such file or directory: b'/Users/MAM/edirect/pytest'

>  at /usr/local/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/os.py:607 in _execvpe
       603│         path_list = map(fsencode, path_list)
       604│     for dir in path_list:
       605│         fullname = path.join(dir, file)
       606│         try:
    →  607│             exec_func(fullname, *argrest)
       608│         except (FileNotFoundError, NotADirectoryError) as e:
       609│             last_exc = e
       610│         except OSError as e:
       611│             last_exc = e
make: *** [test] Error 1

**Comment by turbomam** [comment]
I pulled https://github.com/linkml/schemasheets/releases/tag/v0.1.5 and `make all` does not complain about `~/edirect/pytest` anymore
